src/app/docs/guides/voice-chat/page.md

```diff
@@ -18,6 +18,42 @@ A detailed video and step by step instructions are coming soon, in the meantime,
 we've called out the important parts of the repo above to help you get started
 with it.
 
+## Quickstart to Running the Sample App
+
+```shell
+git clone https://github.com/sublayerapp/rails_llm_voice_chat_example.git
+```
+
+```shell
+bundle install
+```
+
+```shell
+rails db:migrate
+```
+
+```shell
+export OPENAI_API_KEY=your_openai_api_key` 
+```
+
+<!-- should we use something like `.env` instead? -->
```

Comment:
Yeah, that's a great point. I've mostly been doing this stuff locally but now that we're building out project templates/project generators with a CLI, should probably bake in dotenv from the start. Added https://github.com/sublayerapp/sublayer/issues/73 to track this